Gemfile

```diff
@@ -152,6 +152,7 @@ group :test do
   gem 'faker'
   gem 'launchy'
   gem 'selenium-webdriver'
+  gem "webdrivers", '~> 3.0'
```

Comment:
__[RuboCop]__ Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. ([link](https://sider.review/gh/repos/47168038/pulls/537#issue-3998735))

<sub>posted by Sider</sub>